packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-full-upgrade/run-prober-script.js

```diff
@@ -0,0 +1,95 @@
+// to turn on ts-check:
+/* global E */
+
+// import { E } from "@endo/far";
+
+const PROBER_BUNDLE_ID =
+  'b1-4487fd5e18a225a77ca882890d8faae905ca46fda617877f6e1f969e624db3edf0a4f490d735963f025eb3d772840caec157986f4367c0f66385d919f3f5a56c';
+
+console.info('running zcf prober');
+
+const sub = (a, v) => {
```

Comment:
what happens if you import AmountMath?

Reply:
We have a mechanism to support `imports`.  You used it in #7969: `packages/vats/src/proposals/zcf-proposal.js` starts with `import { E } from '@endo/far';`.

But it's a complex mechanism, and `E` is in the environment of a core-eval script even without that import. I tend to do a certain amount of copy-and-paste before I go for the whole `writeCoreProposal` thing.

Reply:
> I tend to do a certain amount of copy-and-paste before I go for the whole writeCoreProposal thing.

@dckc I'm not positive how to interpret this. Should I change something here?